src/Analysis/Ast/Impl/Analyzer/Evaluation/ExpressionEval.cs

```diff
@@ -108,7 +108,7 @@ public ExpressionEval(IServiceContainer services, IPythonModule module)
 
         public void ReportDiagnostics(Uri documentUri, DiagnosticsEntry entry) {
             // Do not add if module is library, etc. Only handle user code.
-            if (Module.ModuleType == ModuleType.User) {
+            if (Module.ModuleType == ModuleType.User || Module.ModuleType == ModuleType.Specialized) {
```

Comment:
Should not be needed. We only report diagnostics in user code.

Reply:
Sure that's my bad.